### PR TITLE
Added in flight operation job update messages

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/node-agent#213

Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
Verified import gluster and ceph cluster flows.
The sample job update status details are as below

```
{

    "action": "get",
    "node": {
        "key": "/nodes/9e3e5e2d-bfc6-430b-b958-2909f25f5424/_jobs/tendrl.node_agent.flows.import_cluster.ImportCluster_8cebe604-c965-4796-9577-93ddcd56c636",
        "dir": true,
        "nodes": [
            {
                "key": "/nodes/9e3e5e2d-bfc6-430b-b958-2909f25f5424/_jobs/tendrl.node_agent.flows.import_cluster.ImportCluster_8cebe604-c965-4796-9577-93ddcd56c636/00000000000000025537",
                "value": "{\"publisher\": \"node_agent\", \"timestamp\": \"2017-02-18T09:38:51.539323+00:00\", \"caller\": {\"function\": \"run\", \"line_no\": 52, \"filename\": \"__init__.py\"}, \"payload\": {\"message\": \"Import cluster job started on node dhcp41-222.lab.eng.blr.redhat.com\"}, \"priority\": \"info\", \"parent_id\": null, \"node_id\": \"9e3e5e2d-bfc6-430b-b958-2909f25f5424\", \"cluster_id\": \"d5f2afab-f188-45e8-9af6-8c587dca097f\", \"request_id\": \"nodes/9e3e5e2d-bfc6-430b-b958-2909f25f5424/_jobs/tendrl.node_agent.flows.import_cluster.ImportCluster_8cebe604-c965-4796-9577-93ddcd56c636\", \"flow_id\": \"2f94a48a-05d7-408c-b400-e27827f4edef\", \"message_id\": \"8dd57047-ec21-456f-9b7a-3969d45a8c9a\"}",
                "modifiedIndex": 25537,
                "createdIndex": 25537
            },
            {
                "key": "/nodes/9e3e5e2d-bfc6-430b-b958-2909f25f5424/_jobs/tendrl.node_agent.flows.import_cluster.ImportCluster_8cebe604-c965-4796-9577-93ddcd56c636/00000000000000025549",
                "value": "{\"publisher\": \"node_agent\", \"timestamp\": \"2017-02-18T09:39:22.927845+00:00\", \"caller\": {\"function\": \"import_ceph\", \"line_no\": 43, \"filename\": \"ceph_help.py\"}, \"payload\": {\"message\": \"Installed storage binaries on node dhcp41-222.lab.eng.blr.redhat.com\"}, \"priority\": \"info\", \"parent_id\": null, \"node_id\": \"9e3e5e2d-bfc6-430b-b958-2909f25f5424\", \"cluster_id\": \"d5f2afab-f188-45e8-9af6-8c587dca097f\", \"request_id\": \"nodes/9e3e5e2d-bfc6-430b-b958-2909f25f5424/_jobs/tendrl.node_agent.flows.import_cluster.ImportCluster_8cebe604-c965-4796-9577-93ddcd56c636\", \"flow_id\": \"2f94a48a-05d7-408c-b400-e27827f4edef\", \"message_id\": \"3de35a78-dd1f-47e0-a5a8-9348a6056f8c\"}",
                "modifiedIndex": 25549,
                "createdIndex": 25549
            },
            {
                "key": "/nodes/9e3e5e2d-bfc6-430b-b958-2909f25f5424/_jobs/tendrl.node_agent.flows.import_cluster.ImportCluster_8cebe604-c965-4796-9577-93ddcd56c636/00000000000000025550",
                "value": "{\"publisher\": \"node_agent\", \"timestamp\": \"2017-02-18T09:39:22.959837+00:00\", \"caller\": {\"function\": \"import_ceph\", \"line_no\": 76, \"filename\": \"ceph_help.py\"}, \"payload\": {\"message\": \"Started ceph integration daemon on node dhcp41-222.lab.eng.blr.redhat.com\"}, \"priority\": \"info\", \"parent_id\": null, \"node_id\": \"9e3e5e2d-bfc6-430b-b958-2909f25f5424\", \"cluster_id\": \"d5f2afab-f188-45e8-9af6-8c587dca097f\", \"request_id\": \"nodes/9e3e5e2d-bfc6-430b-b958-2909f25f5424/_jobs/tendrl.node_agent.flows.import_cluster.ImportCluster_8cebe604-c965-4796-9577-93ddcd56c636\", \"flow_id\": \"2f94a48a-05d7-408c-b400-e27827f4edef\", \"message_id\": \"bcee4e91-463c-478a-954b-1e92ea334057\"}",
                "modifiedIndex": 25550,
                "createdIndex": 25550
            }
        ],
        "modifiedIndex": 25537,
        "createdIndex": 25537
    }
}
```

File: tendrl/node_agent/flows/import_cluster/ceph_help.py [hunk 1]
```diff
@@ -36,7 +52,8 @@ def import_ceph(integration_id):
                    "etcd_connection": tendrl_ns.config.data['etcd_connection'],
                    "tendrl_ansible_exec_file": "$HOME/.tendrl/ceph-integration/ansible_exec",
                    "log_cfg_path":"/etc/tendrl/ceph-integration/ceph-integration_logging"
-                       ".yaml", "log_level": "DEBUG"}
+                       ".yaml", "log_level": "DEBUG",
+                   "logging_socket_path": "/var/run/.tendrl.message.sock"}
```
Comment (nthomas-redhat) [inline tendrl/node_agent/flows/import_cluster/ceph_help.py]: Its /var/run/tendrl ?

File: tendrl/node_agent/flows/import_cluster/gluster_help.py [hunk 2]
```diff
@@ -36,7 +52,8 @@ def import_gluster(integration_id):
                    "etcd_connection": tendrl_ns.config.data['etcd_connection'],
                    "tendrl_ansible_exec_file": "$HOME/.tendrl/gluster-integration/ansible_exec",
                    "log_cfg_path":"/etc/tendrl/gluster-integration/gluster-integration_logging"
-                       ".yaml", "log_level": "DEBUG"}
+                       ".yaml", "log_level": "DEBUG",
+                   "logging_socket_path": "/var/run/.tendrl.message.sock"}
```
Comment (nthomas-redhat) [inline tendrl/node_agent/flows/import_cluster/gluster_help.py]: same as https://github.com/Tendrl/node-agent/pull/215/commits/204ab8e2d45e6e3f9f54ec1a395df6784c4cff98#r101914704

File: tendrl/node_agent/flows/import_cluster/__init__.py [hunk 3]
```diff
@@ -37,13 +39,32 @@ def run(self):
 
                     tendrl_ns.etcd_orm.client.write("/queue/%s" % uuid.uuid4(),
                                                json.dumps(job))
-
-
+        Event(
```
Comment (nthomas-redhat) [inline tendrl/node_agent/flows/import_cluster/__init__.py]: I think you need to add a status for creating the jobs for other nodes in the cluster(line 31)?

File: tendrl/node_agent/flows/import_cluster/ceph_help.py [hunk 4]
```diff
@@ -36,7 +52,8 @@ def import_ceph(integration_id):
                    "etcd_connection": tendrl_ns.config.data['etcd_connection'],
                    "tendrl_ansible_exec_file": "$HOME/.tendrl/ceph-integration/ansible_exec",
                    "log_cfg_path":"/etc/tendrl/ceph-integration/ceph-integration_logging"
-                       ".yaml", "log_level": "DEBUG"}
+                       ".yaml", "log_level": "DEBUG",
```
Comment (nthomas-redhat) [inline tendrl/node_agent/flows/import_cluster/ceph_help.py]: I think we need to add a step for generating the config files

File: tendrl/node_agent/flows/import_cluster/gluster_help.py [hunk 5]
```diff
@@ -36,7 +52,8 @@ def import_gluster(integration_id):
                    "etcd_connection": tendrl_ns.config.data['etcd_connection'],
                    "tendrl_ansible_exec_file": "$HOME/.tendrl/gluster-integration/ansible_exec",
                    "log_cfg_path":"/etc/tendrl/gluster-integration/gluster-integration_logging"
-                       ".yaml", "log_level": "DEBUG"}
+                       ".yaml", "log_level": "DEBUG",
```
Comment (nthomas-redhat) [inline tendrl/node_agent/flows/import_cluster/gluster_help.py]: same as https://github.com/Tendrl/node-agent/pull/215/commits/204ab8e2d45e6e3f9f54ec1a395df6784c4cff98#r101914952

File: tendrl/node_agent/flows/import_cluster/ceph_help.py (hunk 4 above)
Comment (shtripat) [inline tendrl/node_agent/flows/import_cluster/ceph_help.py]: Done. will add one for the same.

File: tendrl/node_agent/flows/import_cluster/ceph_help.py (hunk 1 above)
Comment (shtripat) [inline tendrl/node_agent/flows/import_cluster/ceph_help.py]: Done

File: tendrl/node_agent/flows/import_cluster/gluster_help.py (hunk 5 above)
Comment (shtripat) [inline tendrl/node_agent/flows/import_cluster/gluster_help.py]: Done

File: tendrl/node_agent/flows/import_cluster/gluster_help.py (hunk 2 above)
Comment (shtripat) [inline tendrl/node_agent/flows/import_cluster/gluster_help.py]: Done

File: tendrl/node_agent/flows/import_cluster/__init__.py (hunk 3 above)
Comment (shtripat) [inline tendrl/node_agent/flows/import_cluster/__init__.py]: Done

**Comment by shtripat** [comment]
@Tendrl/tendrl-core plz review

**Pull request merged**